.dir-locals.el

```diff
@@ -0,0 +1,2 @@
+((haskell-mode
+  (haskell-stylish-on-save . t)))
```

Comment:
Isn't this something which should go in a personal config?

I'm not an Emacs user, so I'm not sure about the idioms used here. I'd be a little surprised if a repo had a vim config which executed programs on write.